tests/storages_tests/journal_tests/create_journal.py

```diff
@@ -0,0 +1,51 @@
+"""This script generates assets for testing backward compatibility of `JournalStorage`.
+
+"""
+
+from argparse import ArgumentParser
+import os
+
+import optuna
+
+
+if __name__ == "__main__":
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--storage-url",
+        default=f"{os.path.dirname(__file__)}/assets/{optuna.__version__}.log",
+    )
+    args = parser.parse_args()
+
+    storage = optuna.storages.JournalStorage(optuna.storages.JournalFileStorage(args.storage_url))
+
+    # Empty study
+    optuna.create_study(storage=storage, study_name="single_empty")
```

Comment:
[nit] Could you add a test case to check `single_empty` study is a truly empty like below?

```python
study = optuna.load_study(...)

assert study.directions == ["minimize"]
assert len(study.user_attrs) == 0
assert len(study.trials) == 0
```